feature/visited/src/main/kotlin/uk/govuk/app/visited/VisitedViewModel.kt

```diff
@@ -123,6 +124,7 @@ internal class VisitedViewModel @Inject constructor(
 
     fun onVisitedItemClicked(title: String, url: String) {
         viewModelScope.launch {
+            delay(1000)
```

Comment:
It has crossed my mind we could check and only add this if the user has accessibility enabled?

Reply:
I think it would make more sense for this delay to happen in the composable rather than the view model as the issue it UI related?